x-pack/platform/plugins/shared/security/public/management/api_keys/api_keys_grid/invalidate_provider/invalidate_provider.tsx

```diff
@@ -166,20 +173,71 @@ export const InvalidateProvider: React.FunctionComponent<Props> = ({
         buttonColor="danger"
         data-test-subj="invalidateApiKeyConfirmationModal"
       >
+        {managedRuleKeys.length && (
```

Comment:
```suggestion
        {!!managedRuleKeys.length && (
```

I noticed that on non-managed keys the managed count was being rendered in the dialog.
<img width="411" height="201" alt="Screenshot 2025-11-24 at 14 22 19" src="https://github.com/user-attachments/assets/da52f13c-2af4-4d57-ac11-6b8ad2c1781b" />
<img width="411" height="301" alt="Screenshot 2025-11-24 at 14 22 32" src="https://github.com/user-attachments/assets/2f8fdb2d-c778-47e6-91c2-29d3edb7b7f0" />

Reply:
Pushed in 3d3e861e7e9c0ce8f8014359c737b91cd2893610